sdcm/cluster.py

```diff
@@ -928,7 +928,8 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
                                          scylla_yaml_contents)
 
         p = re.compile('[# ]*hinted_handoff_enabled:.*')
-        scylla_yaml_contents = p.sub('hinted_handoff_enabled: {}'.format(str(hinted_handoff_enabled).lower()), scylla_yaml_contents)
+        scylla_yaml_contents = p.sub('hinted_handoff_enabled: {}'.format(str(not hinted_handoff_disabled).lower()),
```

Comment:
We need to do this only if the test ask to "disable" hinted_handoff, otherwise we should leave it as default and not change anything in the yaml.
This should verify that by default HH works.